src/schema/objects/extensions.yaml

```
@@ -140,6 +140,12 @@
 
     Used by KIT, Yokogawa, and Ricoh MEG systems.
     Successor to the `.sqd` extension for marker files.
+.ngff:
+  name: NGFF
+  description: |
+    A zarr based format, consisting in a series of nested directories.
+
+    Used by human microscopy in particular in order to provide data stream access for very large data.
```

Other directories currently have a trailing `/`. We should follow that convention until/unless it's changed everywhere. Also some changes to the text.

```suggestion
.ngff/:
  name: OME Next Generation File Format
  description: |
    An OME-NGFF file.
    
    OME-NGFF is a [Zarr](https://zarr.readthedocs.io)-based format, organizing data arrays in nested directories.
    This format was developed by the Open Microscopy Environment to provide data stream access to very large data.
```

---

ok, fixed it, and thanks for the suggestions re description text.